### PR TITLE
Update to Moshi 1.2.0.

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

**Comment by cowtowncoder** [comment]
Thanks!

File: src/main/java/com/cowtowncoder/jsonperf/dzone/write/DZoneWriteMoshi.java [hunk 1]
```diff
@@ -40,16 +32,15 @@ public DZoneWriteMoshi()
     @Override
     public int _writeItems(MeasurementPOJO items, OutputStream out) throws Exception
     {
-        OutputStreamWriter w = new OutputStreamWriter(out, "UTF-8");
-        w.write(adapter.toJson(items));
-        w.close();
+        BufferedSink sink = Okio.buffer(Okio.sink(out));
+        adapter.toJson(sink, items);
         return items.size();
```
Comment (JakeWharton) [inline src/main/java/com/cowtowncoder/jsonperf/dzone/write/DZoneWriteMoshi.java:37]: Missing `sink.emit()`? You might leave this method with unflushed bytes.

**Comment by cowtowncoder** [comment]
@swankjesse any objections to adding `emit()` call there? Ideally I'd add verification step which should make it easier to catch potential non-flushing issues by verifying results.

**Comment by swankjesse** [comment]
Yeah, that's a good idea. Emit or flush or close will all work.

**Comment by cowtowncoder** [comment]
Thanks; added close, which also removes compiler warning.

**Pull request merged**